app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,34 @@
+class StatisticsController < ApplicationController 
+
+  def index
+    @cars = current_user.cars.order(created_at: :asc)  
+    
+    @total_repairs = Car.total_repairs
+    
+    @total_chargings = Car.total_chargings
+
+    @total = @total_repairs.merge(@total_chargings) { |_,a,b| a+b }
+
+    @all_car_repairs = []
+    @all_car_chargings = []
+    for car in @cars
```

Comment:
here is a lot of logic and I think most of it could be done by SQL queries
calculation like this should be done elsewhere - model, helper, etc., but not in controller for sure